fiwaremurano/meta/io.murano/Resources/support.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+same_uuid() {
+  curl 'http://169.254.169.254/openstack/latest/meta_data.json' --output /etc/metadata.json
+  if [ -f "/etc/metadata.json" ]
+  then
+    if grep -Po '"uuid":.*?[^\\]",' /etc/metadata.json | grep `cat /etc/fiware-support/uuid`; then
+      echo "Same uuid"
+      return 1
+    fi
+  fi
+  echo "Different uuid VM"
+  return 0
+}
+
+wait_process_finish() {
+  while pidof /sbin/activate_support_account.py >> /dev/null ;
+  do
+    sleep 1
+  done
+}
+
+reload() {
+  echo "Reloading..."
+  rm /etc/fiware-support/uuid
+  /sbin/activate_support_account.py
+}
+
+region='%REGION%'
+
+curl http://130.206.81.64:3000/v1/support/$region/gpgkey --output /etc/fiware-support/gpgbk.pub
+curl http://130.206.81.64:3000/v1/support/$region/sshkey --output /etc/fiware-support/sshbk.pub
+
+if [ -f "/etc/fiware-support/gpgbk.pub" ]
```

Comment:
we do not have style guide for shell scripts? I mean, I think that line 36, 38, 39, 52, 53 should be in the same line.